### PR TITLE
Added option for custom fill for part of some SVGs + story for showing all icons

### PR DESCRIPTION
**Pull request by arorayash**
- Added class `customFill` which can be used to override the fill of an element inside SVG. Adding an extra prop looked like an overkill since the case is not very common.
- Added a story to display all the icons in the IconSystem. What's left is to display the name of those icons in the same story. Since that requires some changes in the Figma Plugin, will be adding it in a different PR.

Screenshot 2019-07-11 at 10 26 40 am

### CODE REVIEW

**Comment by arorayash** [comment]
Are we following any naming convention for class names? @Kushagra8888

**Comment by Kushagra8888** [comment]
@arorayash No, we don't have a naming convention... I hadn't come across the need for haiving classes yet...

File: src/components/Icon/Icon.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+import * as Icons from "./Icons"
+
+const allIcons = Object.values(Icons)
 
 storiesOf("Icon", module)
   .add("Basic Icon", () => <Icon type="oAddCircle" />)
   .add("Icon with color", () => <Icon type="oAddCircle" fill="red" />)
+  .add("All icons", () => {
+    const allIconsDOM = allIcons.map((i, index) => (
+      <div key={index} style={{ margin: 20 }}>
```
Comment (Kushagra8888) [inline src/components/Icon/Icon.stories.tsx]: Can we also add an `alt` attribute on the div?

File: src/components/Icon/Icon.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+        {i({})}
```
Comment (Kushagra8888) [inline src/components/Icon/Icon.stories.tsx]: trivial comment, but JSX would be a but easier to read.

File: src/components/Icon/Icon.stories.tsx [hunk 3]
```diff
@@ -1,7 +1,27 @@
 import React from "react"
 import { storiesOf } from "@storybook/react"
 import Icon from "./index"
+import * as Icons from "./Icons"
+
+const allIcons = Object.values(Icons)
 
 storiesOf("Icon", module)
   .add("Basic Icon", () => <Icon type="oAddCircle" />)
   .add("Icon with color", () => <Icon type="oAddCircle" fill="red" />)
```
Comment (Kushagra8888) [inline src/components/Icon/Icon.stories.tsx]: Can we have one or two stories with knobs for the key attributes like `heght`, `width`, and `color`?

Comment (arorayash) [reply src/components/Icon/Icon.stories.tsx]: Added the knob for size.

File: src/components/Icon/Icon.stories.tsx (hunk 1 above)
Comment (arorayash) [reply src/components/Icon/Icon.stories.tsx]: Why so? Afaik, it's of no use on the div...but acts as placeholder for images when they are being loaded. Am I missing something?

**Pull request merged**